### PR TITLE
Addition of Error Code

### PR DESCRIPTION
**Pull request by joinimran**
Added a missing error code informaiton.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9190

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@nam31 - FYI Please.

File: windows/deployment/update/windows-update-error-reference.md [hunk 1]
```diff
@@ -320,6 +320,7 @@ The PnP enumerated device is removed from the System Spec because one of the har
 | 0x80240042 | `WU_E_UNKNOWN_SERVICE`            | The update service is no longer registered with `AU`.
 | 0x80240043 | `WU_E_NO_UI_SUPPORT`              | There is no support for `WUA UI`.
 | 0x80240FFF | `WU_E_UNEXPECTED`                 | An operation failed due to reasons not covered by another error code.
+| 0x80070422 |                                   | Windows Update service stopped working or not running.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/windows-update-error-reference.md:323]:
```suggestion
| 0x80070422 |                                   | Windows Update service stopped working or is not running.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jaimeo** [comment]
@joinimran does the error code you added not have an associated message?

**Comment by joinimran** [comment]
@jaimeo - There is no error code associated with this. Please see 

image

**Comment by jaimeo** [comment]
OK thanks! Just checking. :^)

**Pull request merged**